src/core/Main.pm

```diff
@@ -45,13 +45,20 @@ my sub MAIN_HELPER($retval = 0) {
     my sub gen-usage() {
         my @help-msgs;
 
+        my $is-win = $*DISTRO.is-win;
+        my $is-insensitive = $is-win or $*DISTRO ~~ /^macosx/;
```

Comment:
> there are both case sensitive and case insensitive variations on HFS

So this line is just wrong?

Reply:
Moreover, this is using `$*DISTRO ~~ /^macosx/` in sink context (use `&&` instead).